snapcraft/storeapi/_client.py

```diff
@@ -11,7 +11,7 @@
 
 # Set urllib3's logger to only emit errors, not warnings. Otherwise even
 # retries are printed, and they're nasty.
-logging.getLogger(requests.packages.urllib3.__package__).setLevel(logging.ERROR)
+logging.getLogger(requests.packages.urllib3.__package__).setLevel(logging.DEBUG)
```

Comment:
stray debugging accidentally left in?

Reply:
I have no idea what you're talking about :innocent: